test/prep-db/mongo.js

```diff
@@ -24,11 +25,18 @@ const dummyVals = {
 				{ id: 'awesomeFollower', url: 'http://localhost:8006' },
 			]
 		}
-	}
-
+	},
 }
 
-if (typeof(module) !== 'undefined') module.exports = dummyVals
+dummyVals['sessions'] = [
```

Comment:
this is definitely not needed, we dropped reading from the `sessions` collection in https://github.com/AdExNetwork/adex-validator-stack-js/commit/752feb140bb35a36704f04fda58344cf2523ad29

Reply:
okay will make the change